### ISSUE TITLE
Add support for multiple ports

### ISSUE THREAD

**Issue by colans** [pfn/keepasshttp#218]
At present it is only possible to run keepasshttp on one port.  It would be great if it could be running on at least two (2) for multiple browser support.

I use several browsers in my Web development work, and it would be really nice to have Passlfox on Firefox and Chromelpass on Chrome both connect to the same KeePass database.

This should also help with issues like #168 .

If this is something that is deemed worthwhile, I'll open corresponding tickets in the browser plug-in projects, Passlfox & Chromelpass.  It will be necessary for the plug-ins to allow port configuration - each one could use a different port to connect to the DB.

**Comment by pfn** [comment]
Huh? This already works as is, don't need multiple ports for multiple
browsers. Multi user keepass is another story, however.

--

**Comment by colans** [comment]
Ah, must be another issue then.  I had it working in Chrome, but after I set it up with Firefox, nothing was happening as if I never installed the extension.  I figured it couldn't use the same communication channel, but maybe there's something else going on.  I tried disabling and re-enabling, Passlfox still does nothing (and there doesn't seem to be any configuration screen).

Thanks for the quick response.
